src/tracker/face_id/face_detector.py

```diff
@@ -94,7 +94,8 @@ def extract_face_from_track(
 
     def extract_face(self, input: torch.Tensor) -> Optional[torch.Tensor]:
         """
-        Returns the box images in the coordinate of the input
+        Returns the box image in the coordinate of the input.
+        Returns None if the number of faces found is not exactly 1.
```

Comment:
don't worry, also a good logic when this function is used for computing fae embedding of a track